primitives/src/channel.rs

```diff
@@ -38,10 +38,14 @@ pub struct ChannelSpec {
     #[serde(default, skip_serializing_if = "Option::is_none")]
     pub min_targeting_score: Option<f64>,
     /// EventSubmission object, applies to event submission (POST /channel/:id/events)
-    pub event_submission: EventSubmission,
+    pub event_submission: Option<EventSubmission>,
     /// A millisecond timestamp of when the campaign was created
-    #[serde(with = "ts_milliseconds")]
-    pub created: DateTime<Utc>,
+    #[serde(
+        default,
+        skip_serializing_if = "Option::is_none",
+        with = "ts_milliseconds_option"
+    )]
+    pub created: Option<DateTime<Utc>>,
```

Comment:
Isn't this required?

Reply:
Not quite

Reply:
Don't we always set it when channel is created?

Reply:
we do, it should not be an option

Reply:
No, we don't https://github.com/AdExNetwork/adex-validator/blob/master/routes/channelCreate.js#L20

When a channel is created is not a much needed detail. What are majorly required is `validUntil` and `withdrawPeriodStart`